src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
@Team it would have been nice if you tried and solve the linter error instead of disabling the linter check, it's advisable to use the linter diable as a last resort